packages/babel-helpers/src/helpers.ts

```diff
@@ -360,9 +360,10 @@ helpers.inheritsLoose = helper("7.0.0-beta.0")`
   }
 `;
 
+// need a bind because https://github.com/babel/babel/issues/14527
 helpers.getPrototypeOf = helper("7.0.0-beta.0")`
   export default function _getPrototypeOf(o) {
-    _getPrototypeOf = Object.setPrototypeOf
+    _getPrototypeOf = Object.setPrototypeOf.bind()
       ? Object.getPrototypeOf
       : function _getPrototypeOf(o) {
           return o.__proto__ || Object.getPrototypeOf(o);
```

Comment:
`.bind()` cannot be added to a condition.

Also it seems better to use `getPrototypeOf` here.

```js
_getPrototypeOf = Object.getPrototypeOf
      ? Object.getPrototypeOf.bind()
      : fun
```

Reply:
thanks! I didn't notice that mistake